doc/taxonomy.tex

```diff
@@ -0,0 +1,116 @@
+\documentclass[12pt,a4paper]{article}
+%\usepackage[french]{babel}
+\usepackage[utf8]{inputenc}
+\usepackage{amsmath}
+\usepackage{amsfonts}
+\usepackage{amssymb}
+\usepackage{eurosym}
+\usepackage[vlined,algoruled]{algorithm2e}
+%\usepackage{algorithm}
+\usepackage[noend]{algpseudocode}
+\usepackage{lscape}
+\usepackage{enumerate}
+\usepackage{tikz}
+\usetikzlibrary{arrows,calc,fit,backgrounds,matrix}
+\usetikzlibrary{shapes,positioning}
+%\usetikzlibrary{patterns,decorations.pathreplacing, decorations.pathmorphing}
+\usetikzlibrary{intersections}
+\usepackage{mathtools}
+
+%\newcommand{\N}{\mathbb{N}}
+%\newcommand{\R}{\mathbb{R}}
+\def\R{{\rm I\!R}}
+\def\N{{\rm I\!N}}
+\def\B{ \{0, 1\} }
+%\newcommand{\pkE}{\check{\mathcal{E}}}
+%\newcommand{\cvE}{\hat{\mathcal{E}}}
+%\newcommand{\elE}{\dot{\mathcal{E}}}
+\newcommand{\pkE}{\mathcal{E}^{\leq}}
+\newcommand{\cvE}{\mathcal{E}^{\geq}}
+\newcommand{\elE}{\mathcal{E}^{=}}
+\newcommand{\elEk}{\mathcal{E}^{=,k}}
+
+%\cev definition : \vec with reversed arrow
+\makeatletter
+\DeclareRobustCommand{\cev}[1]{%
+  \mathpalette\do@cev{#1}%
+}
+\newcommand{\do@cev}[2]{%
+  \fix@cev{#1}{+}%
+  \reflectbox{$\m@th#1\vec{\reflectbox{$\fix@cev{#1}{-}\m@th#1#2\fix@cev{#1}{+}$}}$}%
+  \fix@cev{#1}{-}%
+}
+\newcommand{\fix@cev}[2]{%
+  \ifx#1\displaystyle
+    \mkern#23mu
+  \else
+    \ifx#1\textstyle
+      \mkern#23mu
+    \else
+      \ifx#1\scriptstyle
+        \mkern#22mu
+      \else
+        \mkern#22mu
+      \fi
+    \fi
+  \fi
+}
+\makeatother
+\newcommand{\opp}[1]{\mathrlap{\cev{#1}}\vec{#1}}
+
+\newtheorem{observation}{Observation}
+\newtheorem{proposition}{Proposition}
+\newtheorem{Ass}{Assumption}
+\newenvironment{proof}{\paragraph{Proof:}}{\hfill$\square$}
+
+% \hoffset = -2cm
+% \voffset = -2cm
+% \textheight = 26cm
+% \textwidth = 17cm
+
+
+\title{A taxonomy for routing applications}
+
+\author{Atoptima}
+
+\begin{document}
+\maketitle
+
+Routing applications in logistics come in a rich panel of variations. Where possible we consider the most general case that can be accommodated in our generic data structure; for otherwise, we make restrictive assumptions on the problem variant. So, we consider
+\begin{itemize}
+\item a {\bf static and deterministic} problem as opposed to stochastic or with dynamically incoming data a multi period horizon;
+\item {\bf pure routing} decisions as opposed to integration with driver timetabling, stock management or production scheduling; in particular the framework does not implement restrictions due to driver regulations;
+\item a {\bf single route} or trip per period (the extension to the multiple route case is not straightforward);  but
```

Comment:
Maybe better "a single route or trip per vehicle for the optimization horizon"? 
I understand that you mean by period the "rolling horizon" period for example. ut others use "period of time" as a discrete measure of time which might create confusion.